akka-http/src/main/scala/akka/http/scaladsl/common/StrictForm.scala

```diff
@@ -125,10 +126,10 @@ object StrictForm {
           }
 
         tryUnmarshalToQueryForm.fast.recoverWith {
-          case Unmarshaller.UnsupportedContentTypeException(supported1) ⇒
+          case UnsupportedContentTypeException(supported1, _) ⇒
```

Comment:
if this is required we still have a binary compatibility problem, I think we need the single param unapply or else it could potentially break usage like the old one.

Reply:
isn't binary compatibility checked automatically so nobody guesses about it?

Reply:
Unfortunately there are some cases that are not yet caught by MiMa, and this is one of them: https://github.com/lightbend/migration-manager/issues/40